arbitrator/jit/src/wavmio.rs

```diff
@@ -101,6 +109,38 @@ pub fn read_delayed_inbox_message(mut env: WasmEnvMut, sp: u32) -> MaybeEscape {
     inbox_message_impl(&sp, inbox, "wavmio.readDelayedInboxMessage")
 }
 
+// Reads a hotshot commitment
+fn read_hotshot_commitment_impl(
```

Comment:
Could you please help me understand this a bit better?

Reply:
Yes, essentially what we are doing is validating that espresso batches (which currently map 1-to-1 to arbitrum batches) correspond to a valid hotshot commitment. This function implements a host-io function that reads a hotshot commitment, so that the wasm layer can pass a commitment`ProduceBlock